_rest_routes.inc.php

```diff
@@ -404,5 +404,13 @@
     },
     "GET /fhir/Immunization/:id" => function ($id) {
         return (new FhirImmunizationRestController(null))->getOne($id);
+    },
+    "GET /fhir/Condition" => function () {
+        RestConfig::authorization_check("patients", "med");
```

Comment:
for the 2 immunization routes above this, they should also have this same auth check
```
RestConfig::authorization_check("patients", "med");
```

Reply:
Ok I'll fix those